eth_worker/eth_manager/processor.py

```diff
@@ -462,7 +463,7 @@ def new_transaction_attempt(self, task):
 
         else:
             signature(utils.eth_endpoint('_attempt_transaction'),
-                      args=(task.uuid,)).apply_async(
+                      args=(task.uuid,)).apply(
```

Comment:
`apply_async`->`apply`   
`apply_async` enqueues the job, where `apply` runs them in the local thread. Before this, there was a possible deadlock state where many workers are all waiting for tasks at the bottom of the queue.

Reply:
how's that possible? Doesn't `new_transaction_attempt` just throw the task on the queue and then release the worker?

Reply:
Meant to comment this [here](https://github.com/teamsempo/SempoBlockchain/pull/142#discussion_r402813813), yeah. Think it makes sense to do internal calls like this anyway, just to make the entire transaction happen around the same time.